### PR TITLE
Deprecate shh and bzz

### LINKED ISSUE #3871: deprecate bzz
remove bzz:
- [x] deprecation warning as a console.warn (1.3.4)
- [X] coms of depreciation 
- [ ] removal of package and tests release after warn release (1.3.5)

### LINKED ISSUE #3888: deprecate shh
remove shh:

- [x] deprecation warning as a console.warn (1.3.4)
- [ ] coms of depreciation
- [ ] removal of package and tests release after warn release (1.3.5)

### PR DESCRIPTION
**Pull request by frankiebee**
closes #3871 
closes #3888

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -371,3 +371,7 @@ Released with 1.0.0-beta.37 code base.
 - Bump `elliptic` from `6.5.3` to `6.5.4` for `web3-bzz` (#3940)
 - Bump `elliptic` from `6.5.3` to `6.5.4` for `web3-core-requestmanager` (#3945)
 - Rewrite `web3-eth-iban` in ES6 (#3955)
+
+### Removed
+
+- removed bzz and shh api
```
Comment (Spacesai1or) [inline CHANGELOG.md:381]: So we'd probably want to keep all of the listed changes, besides this one, listed under `1.3.5` for consistency with `1.x`

The `3.x` section should only contain:
```
### Removed

- removed bzz and shh api
```

File: docs/web3-eth-subscribe.rst [hunk 2]
```diff
@@ -78,7 +78,7 @@ clearSubscriptions
 
 Resets subscriptions.
 
-.. note:: This will not reset subscriptions from other packages like ``web3-shh``, as they use their own requestManager.
+.. note:: This will not reset subscriptions from other packages like.
```
Comment (Spacesai1or) [inline docs/web3-eth-subscribe.rst]: No longer a complete sentence, either give another package example, or even remove the line?

File: test/batch.js [hunk 3]
```diff
@@ -47,7 +47,7 @@ describe('lib/web3/batch', function () {
         });
 
         it('should execute batch request for async properties', function (done) {
-
+            // TODO: revisit test
```
Comment (Spacesai1or) [inline test/batch.js]: A TODO

**Review by Spacesai1or — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (frankiebee) [reply CHANGELOG.md:381]: 🤙

**Review by Spacesai1or — APPROVED** [review]

**Pull request merged**